Game/Config/TestOverridesControllerPossessPawnGym.ini

```diff
@@ -0,0 +1,6 @@
+[/Script/UnrealEd.LevelEditorPlaySettings]
```

Comment:
By the way, @improbable-valentyn this will need coordination with @Vix2021 depending on which one goes in first, as you are moving the tests to use generated maps, and this will touch that.
I created `feature/gen-settings` as an experiment to generate these configs automatically using the commandlet. There is one small annoyance, which is that these configs are next to the normal ones, so show up in git and pollute things a bit. Not sure what we want to do with it and whether it's acceptable.
One of the alternatives could be to go at it from the opposite end, and make the test setting override PR check if the map is generated and override settings based on an in-memory string. Opinions welcome.

Reply:
We can have a separate folder for the generated configs if we want a simple solution.
I would need to investigate if it is possible to override settings based on an in-memory string. I would prefer to do this as a separate task so these changes can be committed.

Reply:
Created a draft PR with what I had in mind, just so people can look at the changes: https://github.com/spatialos/UnrealGDK/pull/2912